static/js/app.js

```diff
@@ -276,10 +276,54 @@ $(function() {
             return params;
         },
         setMultipleMarkersIcon: function() {
+            var markerGroups = [];
+            var markerObj;
+            var groupID = 1;
+            var groupsColours = [];
             _.each(this.oms.markersNearAnyOtherMarker(), function(marker) {
                 marker.icon = getIcon("multiple", SEVERITY_VARIOUS);
                 marker.title = 'מספר תאונות בנקודה זו';
+
+                markerObj = {
+                    marker: marker,
+                    group: false
+                };
+                markerGroups.push(markerObj);
             });
+            _.each(markerGroups, function(m){
+                if (!m.group){
+                    m.group = groupID;
+                    groupsColours.push(m.marker.view.model.get('severity'));
```

Comment:
You could extract this marker's severity to a variable, since you're using it later too (groupsColours[groupsColours.length -1])